### ISSUE TITLE
No Preview in 0.78 and 0.79

### ISSUE THREAD

**Issue by HandyManuel** [CCExtractor/ccextractor#285]
Installed 0.79 and there is nothing displayed in the Preview window.
I uninstalled 0.79, installed 0.78 and nothing in Preview window.
Back to 0.77 and Preview window works fine.

One other difference in 0.79 is Target format: (null) compared to 0.77 [Target format: .srt]

0.77 Infomation:

```
C:\Program Files (x86)\CCExtractor\ccextractorwin.exe  --gui_mode_reports  -in=ts -nomyth -autoprogram  -out=srt -ru2 -bom -utf8 -trim --norollup [+input files]

CCExtractor 0.77, Carlos Fernandez Sanz, Volker Quetschke.
Teletext portions taken from Petr Kutalek's telxcc
--------------------------------------------------------------------------
Input: M:\TV\V\SQ\SQ - S02E18 20160219_20002030.ts
[Extract: 1] [Stream mode: Transport]
[Program : 0 ] [Hauppage mode: No] [Use MythTV code: Disabled]

[Timing mode: Auto] [Debug: No] [Buffer input: Yes]
[Use pic_order_cnt_lsb for H.264: No] [Print CC decoder traces: No]
[Target format: .srt] [Encoding: UTF-8] [Delay: 0] [Trim lines: Yes]
[Add font color data: Yes] [Add font typesetting: Yes]
[Convert case: No] [Video-edit join: No]
[Extraction start time: not set (from start)]
[Extraction end time: not set (to end)]
[Live stream: No] [Clock frequency: 90000]
Teletext page: [Autodetect]
Start credits text: [None]
Creating M:\TV\V\SQ\SQ - S02E18 20160219_20002030.srt


-----------------------------------------------------------------

Opening file: M:\TV\V\SQ\SQ - S02E18 20160219_20002030.ts

Analyzing data in general mode
Decode captions from program 1 - MPEG-2 video stream [0x02]  -  PID: 49


New video information found
[1920 * 1080] [AR: 03 - 16:9] [FR: 04 - 29.97] [progressive: no]


XDS: ContentAdvisory: Canadian French Rating: Cette émission peut ne pas convenir aux enfants de moins de 13 ans
  
Number of NAL_type_7: 0
Number of VCL_HRD: 0
Number of NAL HRD: 0
Number of jump-in-frames: 0
Number of num_unexpected_sei_length: 0

CC type 0: 14390 (NTSC line 21 field 1 closed captions)
CC type 1: 2952 (NTSC line 21 field 2 closed captions)
CC type 2: 29917 (DTVCC Channel Packet Data)
CC type 3: 16456 (DTVCC Channel Packet Start)

Total frames time:	  00:20:45:744  (37335 frames at 29.97fps)

Min PTS:				00:00:00:334
Max PTS:				00:20:46:079
Length:				 00:20:45:745

Initial GOP time:	   00:00:00:000
Final GOP time:		 00:20:45:400 +9F
Diff. GOP length:	   00:20:45:400 +9F	(00:20:45:700)

Total user data fields: 74618
HDTV type user data fields: 37309

Done, processing time = 50 seconds
Performance (real length/process time) = 24.91
This is beta software. Report issues to carlos at ccextractor org...
```

0.79 Information:

```
C:\Program Files (x86)\CCExtractor\ccextractorwin.exe  --gui_mode_reports  -in=ts -nomyth -autoprogram  -out=srt -ru2 -bom -utf8 -trim --norollup [+input files]

CCExtractor 0.79, Carlos Fernandez Sanz, Volker Quetschke.
Teletext portions taken from Petr Kutalek's telxcc
--------------------------------------------------------------------------
Input: M:\TV\V\SQ\SQ - S02E18 20160219_20002030.ts
[Extract: 1] [Stream mode: Transport]
[Program : Auto ] [Hauppage mode: No] [Use MythTV code: Disabled]
[Timing mode: Auto] [Debug: No] [Buffer input: Yes]
[Use pic_order_cnt_lsb for H.264: No] [Print CC decoder traces: No]
[Target format: (null)] [Encoding: UTF-8] [Delay: 0] [Trim lines: Yes]
[Add font color data: Yes] [Add font typesetting: Yes]
[Convert case: No] [Video-edit join: No]
[Extraction start time: not set (from start)]
[Extraction end time: not set (to end)]
[Live stream: No] [Clock frequency: 90000]
[Teletext page: Autodetect]
[Start credits text: None]


-----------------------------------------------------------------

Opening file: M:\TV\V\SQ\SQ - S02E18 20160219_20002030.ts

Analyzing data in general mode
Creating M:\TV\V\SQ\SQ - S02E18 20160219_20002030.srt


New video information found
[1920 * 1080] [AR: 03 - 16:9] [FR: 04 - 29.97] [progressive: no]



Total frames time:	  00:20:45:744  (37335 frames at 29.97fps)

Done, processing time = 52 seconds
This is beta software. Report issues to carlos at ccextractor org...
```

**Comment by phanindra707** [comment]
I tried this by removing some comments and added some lines of headers included (from headers of 0.77 version to remove errors) in ccextractor.c of src folder and also added timing header for sync_points etc and got this output.
![kung1](https://cloud.githubusercontent.com/assets/9819126/13643349/55ed757e-e646-11e5-9a0a-2b9291b840c8.png)
![kung2](https://cloud.githubusercontent.com/assets/9819126/13643351/55f9da8a-e646-11e5-9b91-1e3934a8a4e0.png)
![kung4](https://cloud.githubusercontent.com/assets/9819126/13643350/55f7a95e-e646-11e5-9e13-5a6534b5f098.png)
![kung5](https://cloud.githubusercontent.com/assets/9819126/13643352/55fb066c-e646-11e5-888f-755b236ab19a.png)

Sorry as it is in ubuntu

But for performance segmentation fault (commented the performance part) is stdout
![segfault](https://cloud.githubusercontent.com/assets/9819126/13643411/b8f4113c-e646-11e5-82c9-f787ab99a85d.png)

**Comment by HandyManuel** [comment]
Just installed 0.80 and still no preview when extracting .ts file. SRT are identical to 0.79
Getting lots of : 
`XDS Notice: Aspect ratio info, start line=52, end line=232

XDS Notice: Aspect ratio info, active picture height=180, ratio=1.777778`

in activity windows

**Comment by cfsmp3** [comment]
Are you using your own build from source or the windows binaries?

**Comment by HandyManuel** [comment]
I used the Windows binaries

**Comment by cfsmp3** [comment]
Fixed for 0.81
